### PR TITLE
Add link to slack channel

### PR DESCRIPTION
**Pull request by cdrage**
**What kind of PR is this?**

/kind docs

**What does does this PR do / why we need it**:

Adds a link to our new slack channel

**Which issue(s) this PR fixes**:

N/A

**How to test changes / Special notes to the reviewer**:

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -61,7 +61,9 @@ https://asciinema.org/a/wVkVgUrO7PGR5CYBFbHB5fFDn[image:https://asciinema.org/a/
 [[contributing]]
 == Community, discussion, contribution, and support
 
-*Discussions:* For any discussions or help with `odo`, you can use our link:https://groups.google.com/forum/#!forum/odo-users[Google group] or open a link:https://github.com/openshift/odo/issues[GitHub issue].
+*Chat:* Join us on link:https://join.slack.com/t/openshiftdo/shared_invite/enQtODUwMjIzMzE2MjYyLWM3YjlkNTViOGNjZGQ5YjFlMzc2N2RhZjVmMDQ5NWY5ZDQ3NzMzZjYxMTE1MjY5OTVmZjgxZWMyMGRmOTc4Y2I[Slack].
+
+*Discussions:* For any developmental discussions with `odo`, use our link:https://groups.google.com/forum/#!forum/odo-users[Google group].
```
Comment (kadel) [inline README.adoc]: For discussions related to odo development, it should be odo-dev, for getting help and ask questions about using odo it should be odo-users

**Comment by cdrage** [comment]
@kadel updated!

**Pull request merged**